public/app/features/dashboard/components/DashboardSettings/GeneralSettings.test.tsx

```diff
@@ -52,7 +53,7 @@ describe('General Settings', () => {
       const timeZonePicker = screen.getByTestId(selectors.components.TimeZonePicker.containerV2);
       userEvent.click(byRole('combobox').get(timeZonePicker));
       await selectOptionInTest(timeZonePicker, 'Browser Time');
-      expect(props.updateTimeZone).toHaveBeenCalledWith('browser');
+      expect(updateTimeZone).toHaveBeenCalledWith({ timeZone: 'browser' });
```

Comment:
So, to be honest I'm not sure how I feel about testing this here. Making sure selecting a new value from the time-picker updates redux state feels like testing implementation details, but on the other hand there's no change to the DOM or anything that reflects the state change that could otherwise be tested. Maybe better suited for an e2e test? idk